src/Codata/Guarded/Stream.agda

```diff
@@ -0,0 +1,138 @@
+{-# OPTIONS --safe --without-K --guardedness #-}
+------------------------------------------------------------------------
```

Comment:
Missing proper standard library module headers. Also the pragmas should go underneath the module headers.